package.json

```diff
@@ -15,10 +15,11 @@
   "scripts": {
     "lint": "xo generator.js",
     "generate-example": "npm run example-generate",
-    "example-generate": "shx rm -rf tram-one-example && node generator.js tram-one-example",
+    "example-generate": "rm -rf tram-one-example && node generator.js tram-one-example",
     "example-build": "cd ./tram-one-example && npm run build",
     "example-test": "cd ./tram-one-example && npm test",
-    "ci-run": "node tests/circleci-run.js"
+    "ci-run": "node tests/circleci-run.js",
+    "postpublish": "npm run example-generate && rm -rf ./tram-one-example/node_modules && gh-pages -d tram-one-example -b tram-one-example"
```

Comment:
adding a postpublish command that will be called and release our latest template to a branch `tram-one-example`. This branch will be read in the following sandbox: https://codesandbox.io/s/github/Tram-One/tram-one-express/tree/tram-one-example

Reply:
the `rm -rf ./tram-one-example/node_modules` will hopefully be removable once #86 is resolved